src/EthernetServer.cpp

```diff
@@ -6,9 +6,18 @@ extern "C" {
 #include "EthernetClient.h"
 #include "EthernetServer.h"
 
+EthernetServer::EthernetServer()
```

Comment:
If port have a default value defined this one is not required

Reply:
yes I initially wrote a version with single constructor with default param but as arduino API is not using default constructors, I didn't dare to add it, and I am not sure it's 100% same, but I can give an other try to make sure.

Reply:
As you want, it was a suggestion to factorize code because if a later update of the constructor is needed, this avoid to forget one.